modules/misc/xdg-mime.nix

```diff
@@ -46,12 +63,12 @@ in {
       if [[ -w $out/share/mime && -w $out/share/mime/packages && -d $out/share/mime/packages ]]; then
         XDG_DATA_DIRS=$out/share \
         PKGSYSTEM_ENABLE_FSYNC=0 \
-        ${pkgs.buildPackages.shared-mime-info}/bin/update-mime-database \
+        ${getExe cfg.sharedMimeInfoPackage} \
```

Comment:
These are run during the build right?, so this breaks cross. Reported by @colemickens

Reply:
@notalltim 

> I also switched from using `buildPackages` to `pkgs` to improve
cross-compilation compatibility.

This does not improve cross-compilation compatibility, you need to use the package from buildPackages  or an emulator.